### ISSUE TITLE
./configure doesn't check for python

### ISSUE THREAD

**Issue by marmistrz** [LLNL/mpiP#7]
How to reproduce: run `./configure` on a system where `python` is not in the `PATH`.
Then configure succeeds, but make fails.
```
$  make -j 8
python ./make-wrappers.py --xlate --arch=x86_64 --f77symbol symbol_ mpi.protos.txt make-wrappers.py
make: python: Command not found
Makefile:235: recipe for target 'mpiPi_def.h' failed
```

**Comment by surak** [comment]
Plus, it fails with python3

**Comment by cchambreau** [comment]
Python3 support addressed with previous commit [d83e559b39b51464b045dd3fb0f95bc07f9c49a8](https://github.com/LLNL/mpiP/commit/d83e559b39b51464b045dd3fb0f95bc07f9c49a8).  Python test added with commit [18a0a234576cde38724bc0dcfdfdfdc3722e4e7c](https://github.com/LLNL/mpiP/commit/18a0a234576cde38724bc0dcfdfdfdc3722e4e7c)
